src/ui/simulator/toolbox/components/datagrid/renderer/simulation.cpp

```diff
@@ -33,6 +33,16 @@
 
 using namespace Yuni;
 
+// Anonymous namespace: global variable, local scope
+namespace
+{
+enum Antares::Data::TimeSeries mapping[] = {Data::timeSeriesLoad,
```

Comment:
This file is part of GUI, but ok. Why do we not add **Data::timeSeriesRenewable** then ?